### PR TITLE
Lesson classification

### PR DESCRIPTION
**Pull request by Ansper11**
Classification lesson in Jupiter notebook format

- what is  supervised learning and unsupervised learning
- what is K-nearest neighbors
- what is the Euclidean distance
- what is normalization
- Training set and validation set

### CODE REVIEW

**Comment by Ansper11** [comment]
KNN Python code will be uploaded soon

File: Lesson_Classification/Lesson Classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Normalization:\n",
+    "\n",
+    "The next step is normalize our data, but why?\n",
+    "\n",
+    "Consider this example, if we want to determine whether a used car is worth buying based on some of attributes of the car, we can cansider the mileage of the car, the selling price, and the age of the car. However, the maximum age of the car is normally less than 15 years, but its price and mileage can exceed 100,000. There is a problem, when we calculate the distance between each vector, all attributes are treated equally, so if one car is ten years older than the other, we will cansider the 10 years difference have the same importance as 10 dollars difference on the price. This is completely ridiculous. So we need to normalize the data before we can calculate their distance.\n",
```
Comment (Nick-McGee) [inline Lesson_Classification/Lesson Classification.ipynb:90]: Maybe try to make this a Minecraft example :)

Comment (Ansper11) [reply Lesson_Classification/Lesson Classification.ipynb:90]: Thanks, I will try to find one

**Comment by yyht6300a** [comment]
What's your plan about Minecraft world? It's not that easy to make.

**Comment by yyht6300a** [comment]
But it's a great idea still.
